kotlinx-coroutines-debug/test/junit5/CoroutinesTimeoutSimpleTest.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2016-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package junit5
```

Comment:
all tests have a wrong package directive, should be `kotlinx.coroutines.debug.junit5`